js/views/settingsVw.js

```diff
@@ -236,6 +239,21 @@ module.exports = pageVw.extend({
         placeholder_text_multiple: window.polyglot.t('chosenJS.placeHolderTextMultiple')
       });
 
+      var connectedPeers = "<ul>";
+      $.ajax({
+        url: self.serverUrl + "routing_table",
+        success: function(data){
+          self.$('.js-numConnectedPeers').html("<p>" + data.length + "</p>");
```

Comment:
Wrapping the text in a paragraph isn't giving much benefit, and could have unintended consequences from paragraph styles. It might be cleaner to just do
`self.$('.js-numConnectedPeers').text(data.length);`